public/js/webcam.js

```diff
@@ -19,10 +25,17 @@ async function init() {
         tryAgainButton.style.display = "none";
         getTracksButton.style.display = "none";
         loadingDiv.style.display = "none";
+        $("#errorAlert").hide();
+        $('#infoAlert').append('<a class="close alert-close">&times</a>');
         stream = await navigator.mediaDevices.getUserMedia(constraints);
         handleSuccess(stream);
     } catch (err) {
         errorMsgElement.innerHTML = `navigator.getUserMedia.error:${err.toString()}`;
```

Comment:
I think errorMsgElement is null, so this will cause an error, can remove this line completely

Reply:
Sure :)!